src/components/molecules/Alert/Alert.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="alert" :class="`${alertType}`">
+    <div class="alert__title">
+      <div class="alert__icon">
+        <MIcon icon-name="lightbulb" class="alert__icon-fontawesome" />
+      </div>
+      <div v-if="title.length">
```

Comment:
```suggestion
      <div v-if="title">
```
length is not necessary so title is string and not array

Reply:
deleted